p2p_wallet/Services/Storage/KeychainStorage+Extensions.swift

```diff
@@ -32,8 +32,17 @@ extension KeychainStorage: NameStorageType {
 }
 
 extension KeychainStorage: PincodeStorageType {
+    func saveAttempt(_ attempt: Int) {
+        icloudKeychain.set(String(attempt), forKey: pincodeAttemptsKey)
+    }
+
+    var attempt: Int? {
+        Int(icloudKeychain.get(pincodeAttemptsKey) ?? "")
+    }
+
     func save(_ pinCode: String) {
         icloudKeychain.set(pinCode, forKey: pincodeKey)
```

Comment:
I don't think that we need store it in iCloud

Reply:
why not? we're using it to avoid bruteforcing

Reply:
I mean we right now have iCloudKeychain and Localkeychain. 

Assume that you have second phone that synchronises with your first phone. Both of them use different wallets (Wallet A and Wallet B)
The problem is, when attempt reacts 0 on first phone, the second phone also get this value. We shouldn't logout the second phone